docs/usage/networking/shoot_networking.md

```diff
@@ -85,3 +85,17 @@ Number of IPs per podCIDRs: 128
 ```
 
 With the configuration above, a Shoot cluster can at most have **32 nodes** which are ready to run workload in the Pod network.
+
+## Reserved Networks
+
+Some network ranges are reserved for specific use-cases in the communication between seeds and shoots.
+
+| IPv  | CIDR                  | Name               | Purpose                                                                    |
+|------|-----------------------|--------------------|----------------------------------------------------------------------------|
+| IPv4 | 192.168.123.0/24      | Default VPN Range  | Used for communication between seed API server and shoot resources via VPN |
```

Comment:
Maybe we should add that this will be removed from the list once feature gate `NewVPN` is graduated.

Apart from that, it is good to have this section in the documentation.